### PR TITLE
Ctya bignum fix nullptr for openssl 1.0

### PR DESCRIPTION
**Pull request by ctya**
Using latest update for peercoin which also happens to handle this problem well.

https://github.com/peercoin/peercoin/commit/b9ff71634a5d7b262c317363b84d458c33f2c2f9#diff-d9d7c0ac1845d4d4df29b6a75a2a203c

### CODE REVIEW

**Comment by ctya** [comment]
bump

**Comment by ctya** [comment]
This was tested by emfox and looked ok https://github.com/devcoin/core/issues/54

**Comment by emfox** [comment]
yes, I confirm it's good on debian wheezy

File: src/bignum.h [hunk 1]
```diff
@@ -127,7 +128,6 @@ class CBigNum
         return BN_get_word(self);
     }
 
-
```
Comment (develCuy) [inline src/bignum.h]: Mixing Coding style changes with the actual fixes is discouraged, we should open an specific issue for such

**Pull request merged**